fix some param references (#2225)
PX4/PX4-user_guide: en/config_vtol/vtol_quad_configuration.md

Generally back-transition throttle can be set to 0 since forward thrust is not (in most cases) desirable.
If the motor controller supports reverse thrust however, you can achieve this by setting a negative value.

> #### Forward Transition Pusher/Puller Slew Rate

Reviewer: FWIW I _think_ it is better to be precise here. What was there was more "readable" to me - i.e. I understood it without having to read what slew rate means.

Parameter: [VT_PSHER_SLEW](../advanced_config/parameter_reference.md#VT_PSHER_SLEW)

A forward transition refers to the transition from multirotor to fixed-wing mode.

> This is the amount of time in seconds that should be spent ramping up the throttle to the target value (defined by `VT_F_TRANS_THR`).

Reviewer: ```suggestion
The forward transition pusher/puller slew rate is the amount of time in seconds that should be spent ramping up the throttle to the target value (defined by `VT_F_TRANS_THR`).

```

Generally back-transition throttle can be set to 0 since forward thrust is not (in most cases) desirable.
If the motor controller supports reverse thrust however, you can achieve this by setting a negative value.

#### Forward Transition Pusher/Puller Slew Rate

Parameter: [VT_PSHER_SLEW](../advanced_config/parameter_reference.md#VT_PSHER_SLEW)

A forward transition refers to the transition from multirotor to fixed-wing mode.
This is the amount of time in seconds that should be spent ramping up the throttle to the target value (defined by `VT_F_TRANS_THR`).
A value of 0 will result in commanding the transition throttle value being set immediately.
By default the slew rate is set to 0.33, meaning that it will take 3s to ramp up to 100% throttle.

> If you wish to smoothen the throttling up you can reduce it.

Reviewer: I think this reads a little better.

```suggestion
If you wish to make throttling-up smoother you can reduce this value.

```

Start your transition.
It should transition within 50 – 100 meters.
If it doesn’t or it isn’t flying in a stable fashion abort the transition (see below) and land or hover back to the start position and land.

> Try increasing the [transition throttle](#transition-throttle) (`VT_F_TRANS_THR`) value.

Reviewer: @sfuhrer FYI, you'll note I removed the anchor "#transition_throttle" from above the heading and changed the link to use a `-`. This is the default heading anchor format for vuepress and it works better for linking on vuepress than having an explicit anchor.

Generally nowdays I use the vuepress format for headings and link directly. I use the default heading anchor unless it is really long and likely to be linked in a lot of places. 

. The way we do link headings is a little arbitrary - in gitbook they used underscore separators and in vuepress headings get `-` separators by default.

Also consider reducing the transition duration (`VT_F_TRANS_DUR`) if you are not using an airspeed sensor.
If you are using an airspeed sensor consider lowering the transition airspeed but stay well above the stall speed.

As soon as you notice the transition happen be ready to handle height loss which may include throttling up quickly.

> :::warning

Reviewer: FYI, caution not supported in vuepress by default. Decided to just use warning everywhere.

For advanced tuning of the back-transition please refer to the [Back-transition Tuning Guide](vtol_back_transition_tuning.md)

> <span id="aborting_a_transition"></span>

Reviewer: FYI, removed this - nothing links to it.
